docs/features/writing-different-telemetry-types.md

```diff
@@ -30,10 +30,47 @@ Dependencies allow you to track how your external dependencies are doing to give
 
 We provide support for the following dependencies:
 
+- [Azure Service Bus](#measuring-azure-service-bus-dependencies)
 - [Custom](#measuring-custom-dependencies)
 - [HTTP](#measuring-http-dependencies)
 - [SQL](#measuring-sql-dependencies)
 
+### Measuring Azure Service Bus dependencies
+
+Here is how you can report an Azure Service Bus dependency:
+
+```csharp
+var telemetryContext = new Dictionary<string, object>
+{
+    { "Namespace", "azure.servicebus.namespace" }
+};
+
+var durationMeasurement = new Stopwatch();
+
+// Start measuring
+durationMeasurement.Start();
+var startTime = DateTimeOffset.UtcNow;
+
+_logger.LogServiceBusQueueDependency(queueName: "ordersqueue", isSuccessful: true, startTime, durationMeasurement.Elapsed, telemetryContext);
```

Comment:
We should highlight that we have an overload for topic or generic as well

Reply:
Yes, add a note about that.

Reply:
Added suggestion above as well, think both are the sweet spot.